scss/atoms/typography/_typography.scss

```diff
@@ -54,7 +54,7 @@ h6 {
 	font-size: em( 16px );
 	font-weight: 600;
 	line-height: 1;
-	text-transform: uppercase;
+	text-transform: uppercase; // FIXME Is this opinionated?
 }
```

Comment:
I know there was discussion at one point about how we intend to have Chassis be used as a library/framework - if updates to the code could easily be done within Chassis itself - or if the expectation was that any modifications should be done from outside of Chassis, and Chassis's code should be left intact. If the latter - we probably want to make it a variable to avoid a lot of overwriting of styles.   A plus side for making it a variable though is that a designer could better leverage the themeroller in customizing how Chassis looks.